pantheon-bundle/src/main/java/com/redhat/pantheon/model/module/Metadata.java

```diff
@@ -8,6 +8,7 @@
 
 import javax.inject.Named;
 import java.util.Calendar;
+import java.util.Date;
```

Comment:
Again not a big deal here, but can you just revert this file since there are no meaningful changes?

Reply:
Done!